.github/workflows/main.yml

```diff
@@ -140,10 +151,21 @@ jobs:
           yarn set version stable
           yarn add --dev @repo/tsconfig@file:/tmp/artifacts/tsconfig.tgz
           yarn set resolution @inquirer/core@npm:* file:/tmp/artifacts/inquirer-core.tgz
-          yarn set resolution @inquirer/prompts@npm:* file:/tmp/artifacts/inquirer-prompts.tgz
           yarn set resolution @inquirer/figures@npm:* file:/tmp/artifacts/inquirer-figures.tgz
           yarn set resolution @inquirer/ansi@npm:* file:/tmp/artifacts/inquirer-ansi.tgz
+          yarn set resolution @inquirer/external-editor@npm:* file:/tmp/artifacts/external-editor.tgz
           yarn set resolution @inquirer/type@npm:* file:/tmp/artifacts/inquirer-type.tgz
+          yarn set resolution @inquirer/prompts@npm:* file:/tmp/artifacts/inquirer-prompts.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/checkbox@npm:* file:/tmp/artifacts/inquirer-checkbox.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/confirm@npm:* file:/tmp/artifacts/inquirer-confirm.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/editor@npm:* file:/tmp/artifacts/inquirer-editor.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/expand@npm:* file:/tmp/artifacts/inquirer-expand.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/input@npm:* file:/tmp/artifacts/inquirer-input.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/number@npm:* file:/tmp/artifacts/inquirer-number.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/password@npm:* file:/tmp/artifacts/inquirer-password.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/rawlist@npm:* file:/tmp/artifacts/inquirer-rawlist.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/search@npm:* file:/tmp/artifacts/inquirer-search.tgz
+          yarn set resolution @inquirer/prompts/@inquirer/select@npm:* file:/tmp/artifacts/inquirer-select.tgz
```

Comment:
This is what's breaking the e2e test.

Resources:
- https://yarnpkg.com/configuration/manifest#resolutions
- https://yarnpkg.com/cli/set/resolution

```
yarn set resolution <descriptor> <resolution>
```

The `<descriptor>` isn't correct to link the prompt packages required by `@inquirer/prompts` to use the one from the repo.

I'm short on time to figure the exact syntax, but that's the problem. Hopefully it gives you enough to research a bit the issue 🤞🏻